docs/autoplay.md

```diff
@@ -0,0 +1,82 @@
+## Managing Auto Play
+If you would like to start automatically playing your content, you can easily manipulating the player configuration and set the `playback.autoplay` value accordingly.
+
+>### playback.autoplay
+>##### Type: `boolean`
+>##### Default: `false`
+>##### Description: Start playback automatically.
+>If `true`, playback will start automatically when sources will be given to the player. If `false`, a user gesture will be required to start playback.
```

Comment:
If set to `true`, playback will start automatically when sources will be provided to the Player. If set to `false`, a user action will be required to start playback.